NativeApp/src/Emscripten/EmscriptenMain.cpp

```diff
@@ -34,6 +34,7 @@
 std::unique_ptr<Diligent::NativeAppBase> g_pTheApp  = nullptr;
 Diligent::Timer                          g_Timer    = {};
 double                                   g_PrevTime = 0.0;
+const char*                              g_CanvasID = "#canvas";
```

Comment:
Avoid using global variables.
You can pass canvas ID to EventResizeCallback through user data.